Packs/AtlassianJiraServiceManagement/Integrations/AtlassianJiraServiceManagement/AtlassianJiraServiceManagement.py

```diff
@@ -0,0 +1,831 @@
+            'comment': comment
+        }
+        return self._http_request(
+            method='POST',
+            url_suffix=url_suffix,
+            json_data=json_data
+        )
+
+    def send_file(self, object_id, file_path):
+        return self._http_request(
+            method='POST',
+            url_suffix=f'/attachments/object/{object_id}',
+            files={'file': (file_path, open(file_path, 'rb'))}
+        )
+
+    def update_object(self, object_id: str, json_data: Dict[str, Any]):
+        return self._http_request(
+            method='PUT',
+            url_suffix=f'/object/{object_id}',
+            json_data=json_data
+        )
+
+    def delete_object(self, object_id: str):
+        return self._http_request(
+            method='DELETE',
+            url_suffix=f'/object/{object_id}'
+        )
+
+    def remove_file(self, attachment_id):
+        return self._http_request(
+            method='DELETE',
+            url_suffix=f'/attachments/{attachment_id}'
+        )
+
+
+''' HELPER FUNCTIONS '''
+
+
+def pascal_case(s: str) -> str:
+    """ Convert a string to PascalCase. """
+    words = s.split('_')
+    return ''.join(word[:1].upper() + word[1:] for word in words)
+
+
+def convert_keys_to_pascal(objects: List[dict[str, Any]], key_mapping: Optional[dict[str, str]] = None) -> List[dict[str, Any]]:
+    """
+    Convert keys of objects in a list to PascalCase, with optional key mapping.
+
+    Args:
+        objects (list): List of dictionaries (objects) whose keys need conversion.
+        key_mapping (dict): Dictionary containing original keys and expected output keys.
+
+    Returns:
+        list: List of dictionaries with keys converted to PascalCase or as per key_mapping.
+    """
+    if not key_mapping:
+        key_mapping = {}
+
+    converted_objects = []
+
+    for obj in objects:
+        converted_obj = {}
+        for key, value in obj.items():
+            if key in key_mapping:
+                new_key = key_mapping[key]
+            else:
+                new_key = pascal_case(key)
+            converted_obj[new_key] = value
+        converted_objects.append(converted_obj)
+
+    return converted_objects
+
+
+def get_object_attribute_string_type(attribute: Dict[str, Any]) -> str:
+    match attribute['Type']:
+        case 0:
+            return attribute['DefaultType']['name']
+        case 1:
+            return 'Object Reference'
+        case 2:
+            return 'User'
+        case 4:
+            return 'Group'
+        case 6:
+            return 'Project'
+        case 7:
+            return 'Status'
+        case 8:
+            return 'Bitbucket Repository'
+
+
+def get_object_outputs(objects: List[Dict[str, Any]]) -> tuple[List[Dict[str, Any]], List[Dict[str, Any]]]:
+    outputs = []
+    readable_outputs = []
+    for obj in objects:
+        obj_type = obj['ObjectType']['name']
+        output = {k: v for k, v in obj.items() if k != 'ObjectType' and k != 'Avatar'}
```

Comment:
```suggestion
        output = {k: v for k, v in obj.items() if k not in ('ObjectType',  'Avatar')}
```
why this exclusion?

Reply:
Object type is redundant as one could run the object_type_list command and the structure is quite. Avatar is also a large structure and Moishy decided agains it